app/js/lib/foxbox/settings.js

```diff
@@ -5,8 +5,8 @@ import EventDispatcher from './common/event-dispatcher';
 // Prefix all entries to avoid collisions.
```

Comment:
This PR is big enough already so if you want some incremental landing, maybe you can separate the effort for #130 in a first PR. If not, I'm fine with reviewing it as part of this patch.